api_validation/pom.xml

```diff
@@ -57,6 +63,17 @@
             <groupId>org.apache.spark</groupId>
             <artifactId>spark-sql_${scala.binary.version}</artifactId>
             <version>${spark.version}</version>
+            <exclusions>
+                <exclusion>
+                    <groupId>org.apache.curator</groupId>
+                    <artifactId>curator-recipes</artifactId>
+                </exclusion>
+            </exclusions>
+        </dependency>
+        <dependency>
```

Comment:
so I missed this earlier, I think we want to leave this for normal versions and make this specific for cloudera. Really we don't need the cdh profile in this submodule, so lets just remove all the changes from here.